build.gradle.kts

```diff
@@ -285,21 +269,25 @@ signing {
 dependencies {
     implementation("com.intershop.gradle.version:extended-version:3.1.0")
     implementation(gradleKotlinDsl())
-    implementation(kotlin("stdlib-jdk8"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.9.10")
```

Comment:
Since this PR migrates to >= Kotlin 1.8.0, please don't declare `kotlin-stdlib-jdk8` as dependency, but use `kotlin-stdlib` instead: https://kotlinlang.org/docs/whatsnew18.html#updated-jvm-compilation-target